src/smart-components/group/role/group-roles.js

```diff
@@ -61,15 +60,12 @@ const GroupRoles = ({
 
   const actionResolver = () => (
     [
-      ...userIdentity && userIdentity.user && userIdentity.user.is_org_admin ?
-        [
-          {
-            title: 'Remove from group',
-            onClick: (_event, _rowId, role) => {
-              removeRoles(uuid, [ role.uuid ]);
-            }
-          }
-        ] : []
+      {
```

Comment:
Please include `...userIdentity && userIdentity.user && userIdentity.user.is_org_admin ?` in here
```JS
        ...userIdentity && userIdentity.user && userIdentity.user.is_org_admin ?
        [
          title: 'Remove from group',
          onClick: (_event, _rowId, role) => {
            removeRoles(uuid, [ role.uuid ], () => fetchRolesForGroup(pagination)(uuid));
          }
        ]
```